charms/kfp-profile-controller/metadata.yaml

```diff
@@ -11,7 +11,7 @@ resources:
   oci-image:
     type: oci-image
     description: Backing OCI image for kfp-profile-controller
-    upstream-source: python:3.7
+    upstream-source: python:3.11.9-alpine
```

Comment:
Hey @kimwnasptd do we know if by changing the base we'll lose packages needed at runtime? Could you please document the reason to move from plain `python:3.7` to `python:3.11.9-alpine` apart from the CVE fixes?

Reply:
@DnPlas it's not clear to me what you mean with packages needed at runtime.

The source of the image is only a python script and doesn't call any subprocess of the OS
https://github.com/kubeflow/pipelines/blob/2.0.3/manifests/kustomize/base/installs/multi-user/pipelines-profile-controller/deployment.yaml#L15
https://github.com/kubeflow/pipelines/blob/2.0.3/manifests/kustomize/base/installs/multi-user/pipelines-profile-controller/sync.py

Do you have a specific concern in mind?

Reply:
I guess I was asking if you were aware of any potential issues with missing packages since Alpine Linux is a much smaller image than others, meaning it could have less packages than `python:3.7`. It does not look like the `sync.py` script is using anything else, so it looks like we could be fine.

Could you please also write in the description why we went with `python:3.11.9-alpine` and not `python:3.11.9`?

Reply:
@DnPlas I've updated the description

Reply:
@DnPlas ping